examples/label_example.js

```diff
@@ -0,0 +1,23 @@
+const epsagon = require('../src/index.js');
```

Comment:
maybe change this to require('epsagon'), doesn't really matter that it doesn't work cause users will copy paste it.
Maybe have a look at other repos examples and see what is the standard?